spyder/plugins/variableexplorer/widgets/dataframeeditor.py

```diff
@@ -65,6 +69,10 @@
 from spyder.utils.palette import SpyderPalette
 from spyder.utils.qthelpers import keybinding, qapplication
 from spyder.utils.stylesheet import AppStyle, MAC
+if TYPE_CHECKING:
+    from spyder.plugins.variableexplorer.widgets.namespacebrowser import (
```

Comment:
```suggestion

if TYPE_CHECKING:
    from matplotlib.figure import Figure
    from spyder.plugins.variableexplorer.widgets.namespacebrowser import (
```

- Add a blank line for clarity.
- Move the Matplotlib import here.